packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -566,11 +566,11 @@ class CircularProgressIndicator extends ProgressIndicator {
   }) : _indicatorType = _ActivityIndicatorType.material;
 
   /// Creates an adaptive progress indicator that is a
-  /// [CupertinoActivityIndicator] in iOS and [CircularProgressIndicator] in
-  /// material theme/non-iOS.
+  /// [CupertinoActivityIndicator] in iOS & macOS and [CircularProgressIndicator] in
+  /// material theme/non-iOS or macOS.
```

Comment:
Small nit here. With this, there are three places close together that say either iOS & macOS or iOS or macOS.

Can this be changed to "[CupertinoActivityIndicator] in iOS & macOS and [CircularProgressIndicator] in
  /// material theme/non-Apple platforms."

Reply:
> Small nit here. With this, there are three places close together that say either iOS & macOS or iOS or macOS.
> 
> Can this be changed to "[CupertinoActivityIndicator] in iOS & macOS and [CircularProgressIndicator] in /// material theme/non-Apple platforms."

I completely forgot this PR exist, it's updated now